PWGHF/TableProducer/treeCreatorDsToKKPi.cxx

```diff
@@ -223,7 +225,9 @@ struct HfTreeCreatorDsToKKPi {
   Produces<o2::aod::HfCandDsFullPs> rowCandidateFullParticles;
   Produces<o2::aod::HfCandDsLites> rowCandidateLite;
 
-  Configurable<int> decayChannel{"decayChannel", 1, "Switch between decay channels: 1 for Ds->PhiPi->KKpi, 2 for Ds->K0*K->KKPi"};
+  Configurable<int> decayChannelDs{"decayChannelDs", 1, "Switch between decay channels: 1 for Ds->PhiPi->KKpi, 2 for Ds->K0*K->KKPi"};
+  Configurable<int> decayChannelDplusMC{"decayChannelDplusMC", 1, "Switch between decay channels for MC Dplus: 1 for Dplus->PhiPi->KKpi, 2 for Dplus->K0*K->KKPi"};
+  Configurable<bool> FillDplusMC{"fillDplusMC", false, "Switch to fill Dplus MC information"};
```

Comment:
Why only MC? Do you treat Ds and D+ differently in real data?
```suggestion
  Configurable<int> decayChannelDs{"decayChannelDs", 1, "Switch between decay channels: 1 for Ds->PhiPi->KKpi, 2 for Ds->K0*K->KKPi"};
  Configurable<int> decayChannelDplusMc{"decayChannelDplusMc", 1, "Switch between decay channels for MC Dplus: 1 for Dplus->PhiPi->KKpi, 2 for Dplus->K0*K->KKPi"};
  Configurable<bool> fillDplusMc{"fillDplusMc", false, "Switch to fill Dplus MC information"};
```

Reply:
Not addressed

Reply:
Yes, the difference between Ds and D+ is relevant only for MC. In the data, they can be treated equally and are separated later through the invariant-mass fits.